content/en/apps/guides/forms/wealth-quintiles.md

```diff
@@ -0,0 +1,61 @@
+---
+title: "Tracking Wealth Quintiles"
+linkTitle: "Quintiles"
+weight: 
+description: >
+  How to track wealth quintiles on the profile of each family member in the household
+relatedContent: >
+
+---
+
+
+A few things are needed to get the data copied onto the family members. 
+
+1. A report submitted with either `doc.fields.NationalQuintile` or `doc.fields.UrbanQuintile` or both. 
+1. An offline user that has the permission of `can_write_wealth_quintiles`. 
+1. Once the form is submitted the contact will then get the fields `"wealth_quintile_national"` and ` "wealth_quintile_urban"` if they are provided in the form.
+
```

Comment:
Additional context would be helpful here. Also, I think this can be simplified a bit:

```suggestion
Household surveys with questions about the home, possessions, and access to safe drinking water have been used to create equity score and improve the targeting of health services. This guide will cover how quintile information from a household survey can be used in customizing care for individual household members. For example, the equity quintile can be used to increase the number of pregnancy follow-ups for women in specific households, or to display specific notes or questions within patient forms.

For quintile information from a household survey to be used for people in that household, a form about a place must contain `NationalQuintile` and/or `UrbanQuintile` fields at the top level. When that form is submitted all the people belonging to that place will get the corresponding `wealth_quintile_national` and `wealth_quintile_urban` fields. Those fields can then be used by forms, tasks, and contact profiles accordingly.

Note that the `can_write_wealth_quintiles` permission is needed for the user that is submitting the form.
```